cmd/snap-repair/runner_test.go

```diff
@@ -152,13 +179,19 @@ func (s *runnerSuite) TestFetchJustRepair(c *C) {
 	runner := repair.NewRunner()
 	runner.BaseURL = mustParseURL(mockServer.URL)
 
+	r := s.mockNow(c, runner)
+	defer r()
+	t0 := time.Now().UTC().Truncate(time.Minute)
+
 	repair, aux, err := runner.Fetch("canonical", "2", -1)
 	c.Assert(err, IsNil)
 	c.Check(repair, NotNil)
 	c.Check(aux, HasLen, 0)
 	c.Check(repair.BrandID(), Equals, "canonical")
 	c.Check(repair.RepairID(), Equals, "2")
 	c.Check(repair.Body(), DeepEquals, []byte("script\n"))
+
+	c.Check(runner.TLSTime().Before(t0), Equals, false)
```

Comment:
Maybe we could make this a tiny bit more descriptive by adding a small helper, something has a name like: `s.checkTLSTimeGotUpdate()` (or similar) which would do the `t0 := time.Now()` internally. With the other suggested name change it would be something like:
```
r := s.simulateTimeNowBroken()
...
c.checkTimeNoLongerBroken()
```
(feel free to pick better names of course :)